### PR TITLE
MuCAI Blog Post - Update about Prof. Eugene 

### PR DESCRIPTION
**Pull request by rjgsousa**
Blog entry about MuCAI Call for Papers and Professor Eugene talk.

@cajosantiago and @davidfsemedo can you review?

### CODE REVIEW

**Review by cajosantiago — CHANGES_REQUESTED** [review]
After "updating the keynote speaker: Professor Eugene Agichtein", we should provide a brief description of his profile / biography / track record, highlighting why he is an exceptional guest and why everyone should attend the event.

Source for inspiration (taken from his home page):
Eugene Agichtein is a Winship Associate Professor in the Computer Science department at Emory University. Dr. Agichtein founded and leads the Emory Intelligent Information Access Laboratory (IR Lab). Eugene's general research interests are in web search and information retrieval, conversational search, and more generally in text and data mining, social media analysis, and human-computer interaction. Eugene received a Ph.D. in Computer Science from Columbia University, and a B.S. in Engineering from The Cooper Union. Eugene was a Sloan Research Fellow, a past member of the DARPA Computer Science Study Group, and a recipient of four best paper awards from SIGMOD, SIGIR, WSDM, and ICDE conferences, and test-of-time awards from WSDM and SIGIR. His work has been supported by grants from DARPA, IARPA, the National Institutes of Health, and the National Science Foundation, and by gifts and grants from industry, including Amazon, Google, Home Depot, HP Labs, Microsoft, and Yahoo! Labs. Since January 2019, Dr. Agichtein has been a part-time "Amazon Scholar" (Principal Scientist) at Amazon Alexa.

**Review by davidfsemedo — COMMENTED** [review]
I agree with @cajosantiago comment about adding the biography of the keynote speaker. 

Also, is the talk topic already defined? If so, we could add something like "Professor Eugene ... will talk about XXX and explain how YYY.

**Comment by rjgsousa** [comment]
> I agree with @cajosantiago comment about adding the biography of the keynote speaker. 
> 
> Also, is the talk topic already defined? If so, we could add something like "Professor Eugene ... will talk about XXX and explain how YYY.

Thanks for the feedback,  @davidfsemedo 
Let me try to find out which topic he will talk about.

**Comment by rjgsousa** [comment]
> After "updating the keynote speaker: Professor Eugene Agichtein", we should provide a brief description of his profile / biography / track record, highlighting why he is an exceptional guest and why everyone should attend the event.
> 
> Source for inspiration (taken from his home page):
> Eugene Agichtein is a Winship Associate Professor in the Computer Science department at Emory University. Dr. Agichtein founded and leads the Emory Intelligent Information Access Laboratory (IR Lab). Eugene's general research interests are in web search and information retrieval, conversational search, and more generally in text and data mining, social media analysis, and human-computer interaction. Eugene received a Ph.D. in Computer Science from Columbia University, and a B.S. in Engineering from The Cooper Union. Eugene was a Sloan Research Fellow, a past member of the DARPA Computer Science Study Group, and a recipient of four best paper awards from SIGMOD, SIGIR, WSDM, and ICDE conferences, and test-of-time awards from WSDM and SIGIR. His work has been supported by grants from DARPA, IARPA, the National Institutes of Health, and the National Science Foundation, and by gifts and grants from industry, including Amazon, Google, Home Depot, HP Labs, Microsoft, and Yahoo! Labs. Since January 2019, Dr. Agichtein has been a part-time "Amazon Scholar" (Principal Scientist) at Amazon Alexa.

Awesome. This is very helpful. Thanks so much @cajosantiago !

File: content/english/blog/blog-post-20210721.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+bg_image: "images/featue-bg.jpg"
+categories: ["Conference"]
+tags: ["Conference"]
+draft: true
+type: "post"
+---
+
+As previously announced, we are organizing a Workshop On Multimedia Conversational AI at ACM Multimedia 2021 ( [original blog post](https://ifetch-chatbot.github.io/blog/blog-post-20210504/)).
+
+In this blog post, we update the keynote speaker: Professor [Eugene Agichtein](https://scholar.google.com/citations?hl=en&user=3BX3vWcAAAAJ&view_op=list_works&sortby=pubdate). We are delighted to have Professor Eugene giving a talk at MuCAI. Keep tuned for more updates.
```
[the remark this answers is not in the document]
Comment (rjgsousa) [reply content/english/blog/blog-post-20210721.md]: fixed in 5db43a5 thx

**Review by cajosantiago — APPROVED** [review]

**Pull request merged**